models/user/index.js

```diff
@@ -414,6 +414,37 @@ class User extends Model {
     return output;
   }
 
+
+    /**
+   * Find [limit] new users (newlycreated and veryfied)
+   * who has at least [similarTagsNumber] similar tags to mine
+   * @param {int, int} limit, similatTagsNumber
+   * @returns {Promise<Object>}
+   */
+  static async findNewUsersWithMyTags(myUsername, limit, commonTagsNumber) {
```

Comment:
common tags number is a really neat idea! TODO on the standard usersWithMyTags, too. :)